### DISCUSSION TITLE
2023/show-path-column/

### DISCUSSION THREAD

**Discussion by giscus** [mirontoli/chuvash-eu-comments#2] [General]
# 2023/show-path-column/

PnP.PowerShell is just the tool you need to show the Path column in a SharePoint document library view. Scroll down to see the script, or read on to learn more.
An introduction Do you still have folders in SharePoint? Well, you are not alone. Remember the old dream of relying on metadata only? Forget it! As a matter of fact, folders are still a popular way of organizing files in SharePoint.

https://chuvash.eu/2023/show-path-column/

**Comment by carenggli** [comment]
Hi Anatoly
Thank you for your post. I just found another solution with JSON formatting (see: https://techcommunity.microsoft.com/t5/sharepoint/displaying-doclib-file-url-path-in-column/m-p/3876909).
In my case I have a library with only one level of folders, so my expression to show nothing but the folder looks like this:
```json
{
  "$schema": "https://developer.microsoft.com/json-schemas/sp/v2/column-formatting.schema.json",
  "elmType": "div",
  "txtContent": "=removeFrom(split(replace([$FileRef],'/sites/[MySharePointSite]/[MySharePointLibrary]/',''), '/'), [$FileLeafRef])"
}
```

Hope it helps :-)

**Reply by mirontoli** [reply]
This is nice. Thanks for sharing!
